### PR TITLE
Update the-difference-between-null-undefined-and-nan.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
remove `function` from pq as it wasn't taught prior to this question

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md?ref=1e0a50347e2f7f86513458d77cfbe5ee71287792)

File: javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md [hunk 1]
```diff
@@ -60,11 +60,7 @@ isNaN("Twenty Three") // true (it's not a number)
 What does the following code output?
 
 ```javascript
-function foo(num){
-  return num / 2;
-}
-
-console.log(foo("sum"));
+console.log("sum" / 2);
```
Comment (kapnobatai137) [inline javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md:63]: @Stefan-Stojanovic what do you think about replacing "sum" with "1"?

```suggestion
console.log("1" / 2);
```

**Review by kapnobatai137 — APPROVED** [review]
lgtm :+1:

File: javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md]: It is better, I just tried to do a fast and simple change

**Review by kapnobatai137 — APPROVED** [review]
:rocket:

**Pull request merged**